lib/views/settings/settings.dart

```diff
@@ -40,23 +40,15 @@ class _ThemePicker extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final themeMode = context.watch<AppTheme>().state;
-    final style = TextStyle(color: context.colorScheme.onBackground);
     return SegmentedButton<ThemeMode>(
-      style: SegmentedButton.styleFrom(
-        side: const BorderSide(style: BorderStyle.none),
-        backgroundColor: context.lightDark(Colors.white54, Colors.black54),
-      ),
```

Comment:
Originally I had styled the button here, but I think it's better to do it in the [theme data](https://github.com/theheartcenter-one/thc/blob/survey-screens/lib/models/theme.dart#L81)—that way, it's consistent across the app without any extra effort!